pysat/instruments/__init__.py

```diff
@@ -11,6 +11,7 @@
            'dmsp_ivm', 'icon_ivm', 'icon_euv', 'icon_fuv', 'icon_mighti',
            'iss_fpmu',  'jro_isr', 'omni_hro', 'pysat_sgp4', 'rocsat1_ivm',
            'sport_ivm', 'superdarn_grdex', 'supermag_magnetometer',
-           'sw_dst', 'sw_kp', 'sw_f107', 'timed_see', 'ucar_tiegcm', ]
+           'sw_dst', 'sw_kp', 'sw_f107', 'timed_saber', 'timed_see',
```

Comment:
Nice. TIMED SABER needs to be added to the official docs as well. docs/supported_instruments.rst  I think.

Reply:
added